__tests__/app.test.js

```diff
@@ -223,3 +223,33 @@ describe('patchArticleById', () => {
     .expect(404)
   })
 })
+
+describe('postComment', () => {
+  test('202: /api/articles/:article_id/comments returns the posted comment and inserts it into comments table', () => {
+    return request(app)
+    .post('/api/articles/2/comments')
+    .send({username: 'lurker',
+            body: "hi this is a test comment"})
+    .expect(202)
+    .then((result) => {
+      expect(result.body).toEqual({author: 'lurker',
+      body: "hi this is a test comment", 
+      article_id: 2,
+      comment_id: 19,
+      created_at: expect.any(String),
+      votes:0
+  }) 
```

Comment:
If this is a POST and something is being created i would say this is a 201 - created rather than a 202 - accepted.